ui-admin/src/user/CreateUserModal.tsx

```diff
@@ -22,16 +23,12 @@ const CreateUserModal = ({ onDismiss, portals, userCreated }:
   const selectedPortalOpt = portalOpts.find(portalOpt => portalOpt.value === portalShortcode)
 
   const createUser = async () => {
-    setIsLoading(true)
-    try {
+    doApiLoad(async () => {
```

Comment:
Ah, interesting. Unlike in `PreEnrollView`, it looks like this use of `doApiLoad` eliminates the need to `catch` and `Store.addNotification` here.